### PR TITLE
add an example page

### PR DESCRIPTION
**Pull request by stephenkingsley**
add an example page, let the test and bug fix more and more convenient and fast.

when run `npm run dev`, it can open the `localhost:3000` to see the page. and when you change some file, they will show the new page.

### CODE REVIEW

**Comment by stephenkingsley** [comment]
yeah~ I will fix it

**Comment by stephenkingsley** [comment]
may u review my code?

**Comment by stephenkingsley** [comment]
ok, I will fix it now!

File: package.json [hunk 1]
```diff
@@ -11,14 +11,16 @@
     "build:lib": "babel --out-dir lib src",
     "prebuild:umd": "rm -rf dist/*",
     "prebuild:dist": "rm -rf dist/*",
+    "build:dev": "rollup -c",
```
Comment (mxstbr) [inline package.json]: I'm not sure we need a separate command – with this example, I'd expect to be able to switch out `styled-components.js` with `styled-components.min.js` in the `index.html` and to test my changes in minfied mode, right?

Comment (stephenkingsley) [inline package.json]: yeah, I agree. I think output `styled-components.js` and `styled-components.min.js` is cost more time. so I want just output one file. I will change it.

**Comment by stephenkingsley** [comment]
may you review again please?

**Comment by mxstbr** [comment]
Let's fix the no-tests thing to not include `example/`, same for the changelog thing!

**Comment by mxstbr** [comment]
How awesome is that! Danger doesn't comment again it edits the previous comment! 😱 😱 😱

**Comment by orta** [comment]
Hah - yep, fav feature of mine - if you have no warnings ( or stickies ) she'll also delete the message

**Pull request merged**